docs/networking/networking-basics.md

```diff
@@ -0,0 +1,520 @@
+# Networking Basics
```

Comment:
Meta question: How useful is it to have a page on a self hosting wiki which is, arguably, going into some pretty low level networking fundemental.

Besides the different between public/private V4 address space, very little has come up for me when self-hosting.